bin/MadGraph5_aMCatNLO/cards/production/13TeV/Wp_To_TpBottom_BpTop_tHb/WpToBpT_Wp1500Nar_Bp1200NarRH_Ht/WpToBpT_Wp1500Nar_Bp1200NarRH_Ht_proc_card.dat

```diff
@@ -0,0 +1,19 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
```

Comment:
can you please remove the "import model sm" line since it will not have any effect?